go/vt/vtgate/planbuilder/operators/delete.go

```diff
@@ -36,3 +36,8 @@ func (d *Delete) Introduces() semantics.TableSet {
 
 // ThisIsAnOperator implements the Operator interface
 func (d *Delete) ThisIsAnOperator() {}
+
+func (d *Delete) Clone(inputs []Operator) Operator {
+	checkSize(inputs, 0)
+	return d
+}
```

Comment:
We don't clone delete at all?

Reply:
`Clone` is used for trying different alternatives of plans. these three ops are not really optimised, so there was no need to do an actual Clone. I could implement something to have it there, so it doesn't look so weird, but it's not really needed

Reply:
actually, it looks like all three structs have OK clone methods. it probably got fixed in later commits